src/test/java/com/jakewharton/rxrelay/PublishRelayTest.java

```diff
@@ -42,15 +42,13 @@
 
     relay.call("one");
     relay.call("two");
-    relay.call("three");
 
     @SuppressWarnings("unchecked") Observer<String> anotherObserver = mock(Observer.class);
     relay.subscribe(anotherObserver);
 
-    relay.call("four");
+    relay.call("three");
 
     assertCompletedObserver(observer);
-    // todo bug?            assertNeverObserver(anotherObserver);
```

Comment:
In RxJava it looks like this is no longer commented out so we should pull in this check instead of deleting it. If you don't want to bother just leave the comment and I'll do it in a follow-up.